sklearn/model_selection/tests/test_search.py

```diff
@@ -1,20 +1,21 @@
 """Test the search module"""
 
+from collections import Iterable, Sized
 from sklearn.externals.six.moves import cStringIO as StringIO
 from sklearn.externals.six.moves import xrange
 from itertools import chain, product
 import pickle
 import sys
 from types import GeneratorType
 import re
+import warnings
 
 import numpy as np
 import scipy.sparse as sp
 import pytest
 
 from sklearn.utils.fixes import sp_version
 from sklearn.utils.fixes import PY3_OR_LATER
-from sklearn.utils.fixes import _Iterable as Iterable, _Sized as Sized
```

Comment:
Why? That's necessary to avoid warnings in Python 3.7

Reply:
I did not change that, my code came directly from the branch I was working on (which is an old version of master).